vibrateonstart/obj/VibrateOnStart.bundle/VibrateOnStart-iPad.plist

```diff
@@ -221,22 +221,6 @@
 		<dict>
 			<key>cell</key>
 			<string>PSGroupCell</string>
-			<key>footerText</key>
```

Comment:
I have removed the Paypal button. Why? Because Paypal take 0.25 € cts in France for each transactions. So if someone give me 0.10 € cts, I will lost 0.15 € cts. So I don't know if a donation button is a good idea...